src/Miningcore/Api/Controllers/PoolApiController.cs

```diff
@@ -448,49 +447,6 @@ public async Task<GetPoolResponse> GetPoolInfoAsync(string poolId)
         return response;
     }
 
-    [HttpGet("{poolId}/miners/{address}/balancechanges")]
```

Comment:
Can we keep these APIs and use cosmos to return data?

Reply:
we removed these APIs awhile back, when we deleted that table.  that is a good idea tho.  let's add a new story.